### DISCUSSION TITLE
Help with local SSL (own CA)

### DISCUSSION THREAD

**Discussion by overfl0w72** [dani-garcia/vaultwarden#4429] [Q&A]
Hello everyone,
I just don’t know what to do anymore. I have been trying to install Vaultwarden on my Proxmox 7.x host for some time. I have created a Debian11 container with Docker Compose and installed Vaultwarden. When I call the local IP address of the host, I also get to the Vaultwarden WEBGUI. Now I want to install a local self-signed SSL certificate, which I created with my local CA. My question is, where do I save this and where do I specify the path? I can’t use a reverse proxy because of DS-Lite and I don’t want Cloudflare. I would really appreciate detailed instructions or a tip. Thank you.

Unfortunately, the following instructions do not help me:
https://github.com/dani-garcia/vaultwarden/wiki/Private-CA-and-self-signed-certs-that-work-with-Chrome

**Comment by stefan0xC** [comment]
Have you read https://github.com/dani-garcia/vaultwarden/wiki/Enabling-HTTPS#via-rocket ?

**Comment by Shejken** [comment]
I had some trouble with this as well but this is how I got it working:
```
root@redacted:/home/redacted/containers/vaultwarden# ls ssl
keys  VaultWarden_20240322.crt  VaultWarden_20240322_key.crt
root@redacted:/home/redacted/containers/vaultwarden# docker run -d --name bitwarden   -e ROCKET_TLS='{certs=ssl/VaultWarden_20240322.crt,key=ssl/VaultWarden_20240322_key.crt}'   -v ssl/:/ssl/   -v /vw-data/:/data/   -p 443:80   vaultwarden/server:latest
```

**Reply by stefan0xC** [reply]
Shouldn't it be `-v ssl/:/ssl/` (relative to the current directory) instead of `-v /ssl/:/ssl/`? 🤔

**Reply by Shejken** [reply]
Of course, thanks.
I've edited my post.
